scripts/spack_test/CMakeLists.txt.in

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.16) 
+project(kokkos_spack_test CXX)
+find_package(Kokkos REQUIRED)
+
+set(SRC_NAME_LIST "@SRC_NAME_LIST@")
+set(BIN_NAME_LIST "@BIN_NAME_LIST@")
+set(CMAKE_CXX_COMPILER ${Kokkos_CXX_COMPILER})
```

Comment:
Oh wait. This does absolutely nothing. At this point, the `CXX` in project already set the C++ compiler, you cannot override it locally.

Reply:
Resolved for now on Slack.